foreFinder/fore-finder-fe
# Player list

## Is this a fix or a feature?
Feature

## What is the change?
Introduction of PlayerList component which displays community/friends in a sidebar on desktop, and a separate page on smaller screensizes (<1024)

## What does it fix?
Users can add and remove friends from their relationships.

## Where should the reviewer start?
Fetch and checkout the playerList branch

## How should this be tested?
Add friend; remove friend; confirm responsiveness; confirm redirect

closes #39 
closes #22
closes #23
